.changeset/yellow-lions-enjoy.md

```diff
@@ -0,0 +1,45 @@
+---
+'@sku-lib/create': major
+---
+
+Add new standalone create-sku CLI package
+
+Introduces a new `@sku-lib/create` package that provides a standalone CLI for scaffolding new sku projects. This package replaces the functionality of `sku init` with a dedicated tool optimized for project creation. The existing `sku init` command is expected to be deprecated in a future release.
```

Comment:
I'd say we deprecate `sku init` and guide users towards the create package immediately after this PR. Don't see a reason not to do that sooner rather than later.

```suggestion
Introduces a new `@sku-lib/create` package that provides a standalone CLI for scaffolding new sku projects. This package replaces the functionality of `sku init` with a dedicated tool optimized for project creation. The existing `sku init` command is now deprecated and will be removed in a future release.
```